scripts/record_deployments.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python
+
+import os
+import sys
+from argparse import ArgumentParser, RawDescriptionHelpFormatter
+from datetime import datetime
+
+import requests
+from requests.auth import HTTPBasicAuth
+
+# API keys and IDs are set with environment variables rather than on the
+# command line, to avoid displaying the key in logs
+NR_API_KEY_NAME = "NEW_RELIC_API_KEY"
```

Comment:
@limed We are gonna need to get API keys (at least for Speedcurve) before we can merge this.

- [ ] @stevejalim Ticket up sourcing of API keys

Reply:
@atopal Could you get hold of a Speedcurve API key off (off a service account?) please?

Reply:
@stevejalim I thought we had that already, but I sent you the key via Slack